surelock/src/main/java/com/smashingboxes/surelock/Surelock.java

```diff
@@ -497,4 +459,142 @@ private void setEncryptionIv(byte[] encryptionIv) {
         return storage.get(KEY_INIT_IALIZ_ATION_VEC_TOR);
     }
 
+    public static class SurelockDialogBuilder {
```

Comment:
I think we can just call this `Builder`. That way when devs initialize it, they just do `new Surelock.Builder(...)`